doc/_resources/templates/document.html

```diff
@@ -58,7 +58,7 @@
 {{end}}
 
 {{define "head"}}
-	{{if (or (not .Content) .ContentVersion)}}<meta name="robots" content="noindex">{{end}}
+	{{if (or (contains .Content.Path "code_insights") (or (not .Content) .ContentVersion))}}<meta name="robots" content="noindex">{{end}}
```

Comment:
```suggestion
	{{if (or (contains .Content.Path "insights") (or (not .Content) .ContentVersion))}}<meta name="robots" content="noindex">{{end}}
```

Caught this, my fault for not catching sooner – Amie responded we need to unindex dev docs too and those are /insights/ https://docs.google.com/document/d/1RHcxSRNRq0uGratOlAYLmO2Aa3CkVJgaPAZk58w15zE/edit?disco=AAAAULI_Pic